src/librustc_target/spec/windows_gnu_base.rs

```diff
@@ -20,9 +20,9 @@ pub fn opts() -> TargetOptions {
     late_link_args.insert(
         LinkerFlavor::Gcc,
         vec![
+            "-lmsvcrt".to_string(),
             "-lmingwex".to_string(),
             "-lmingw32".to_string(),
```

Comment:
Might be worth adding a comment here explaining the order?

Reply:
That order is trial and error that *happens to work right now*. I don't expect it will to last for long.
Not an assuring comment 😆

Reply:
Not assuring, but still useful to know for anyone reading/editing this. :)

Reply:
Opened https://github.com/rust-lang/rust/pull/73241